intro-HPC/ch_troubleshooting.tex

```diff
@@ -64,3 +64,35 @@ \section{Warning message when first connecting to new host}
 \else
 \firsttimeconnection
 \fi
+
+\ifusinglmod
```

Comment:
This is really `/ifgent`, see the mention of `hpc@ugent.be` in the error message (we customize that).

The `ifusinglmod` may come in handy for #192 so leave it there